### PR TITLE
Add options on Import transactions screen to mark transactions as cleared/uncleared

### PR DESCRIPTION
**Pull request by kstockk**
Added additional option on import transactions screen to mark the transactions as cleared/uncleared

image

### CODE REVIEW

File: packages/desktop-client/src/components/modals/ImportTransactions.js [hunk 1]
```diff
@@ -584,6 +584,8 @@ export default function ImportTransactions({ modalProps, options }) {
 
   let [parseDateFormat, setParseDateFormat] = useState(null);
 
+  let [clearOnImport, setClearOnImport] = useState(true);
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/modals/ImportTransactions.js:587]: **💬 suggestion:** ‏could we instead set this to `false` by default? I wouldn't want to break the mental model existing users have when doing manual imports.
```suggestion
  let [clearOnImport, setClearOnImport] = useState(false);
```

**Review by MatissJanis — COMMENTED** [review]
Other than the one comment, this looks very good! Thanks! 👏

File: packages/desktop-client/src/components/modals/ImportTransactions.js (hunk 1 above)
Comment (kstockk) [reply packages/desktop-client/src/components/modals/ImportTransactions.js:587]: Thanks for reviewing. I believe the default behaviour at the moment on import is to mark the transactions as cleared

Comment (MatissJanis) [reply packages/desktop-client/src/components/modals/ImportTransactions.js:587]: Yes, you're actually correct. Thanks for correcting me!

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**